lisp_ease/src/queries/queries.lisp

```diff
@@ -0,0 +1,230 @@
+(in-package :le)
```

Comment:
Rename ``basic-queries.lisp`` into something like ``utility-queries.lisp`` and put all the used bigish ones from basic into this file. Or maybe call it ``test-queries.lisp``.